server/controllers/finance/reports/collection_capacity/report.handlebars

```diff
@@ -0,0 +1,54 @@
+{{> head title="REPORT.COLLECTION_CAPACITY.TITLE"}}
+
+<body>
+
+<div class="container">
+  {{> header}}
+
+  <!-- body  -->
+  <div class="row">
+    <div class="col-xs-12">
+
+      <!-- page title  -->
+      <h3 class="text-center text-uppercase">
+        <strong>{{translate 'REPORT.COLLECTION_CAPACITY.TITLE'}}</strong> <br>
+        <small>{{account.number}} - {{account.label}}</small><br>
+        <small>{{date dateFrom}} - {{date dateTo}}</small>
+      </h3>
+
+      <table class="table table-striped table-condensed table-report table-bordered">
+        <thead>
+          <tr>
+            <th>{{translate 'TABLE.COLUMNS.DATE'}}</th>
+            <th>{{translate 'REPORT.COLLECTION_CAPACITY.REGISTRATIONS'}}</th>
+            <th>{{translate 'REPORT.COLLECTION_CAPACITY.INVOICES'}}</th>
+            <th>{{translate 'REPORT.COLLECTION_CAPACITY.AVERAGE_COST'}}</th>
+            <th>{{translate 'REPORT.COLLECTION_CAPACITY.PAYMENTS'}}</th>
+            <th>{{translate 'REPORT.COLLECTION_CAPACITY.RECOVERY_CAPACITY'}}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {{#each rows as | row |}}
+          <tr>
+            <td>{{date row.invoice_date}}</td>
+            <td style="width: 10em;" class="text-right">{{row.registrations}}</td>
+            <td class="text-right">{{debcred row.total_invoiced metadata.enterprise.currency_id}}</td>
+            <td class="text-right">{{debcred row.avg_cost metadata.enterprise.currency_id}}</td>
+            <td class="text-right">{{debcred row.total_paid metadata.enterprise.currency_id}}</td>
+            <td 
+              style="width: 15em;"
+              {{#gt row.recovery_capacity 0.7}} class="bg-success text-success text-right" {{/gt}}
+              {{#between row.recovery_capacity 0.6 0.7}} class="bg-warning text-warning text-right" {{/between}}
```

Comment:
It looks like all of these have `text-right` in them.  Let's just move it out of this chain.